test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Alias.Tests.ps1

```diff
@@ -44,40 +44,92 @@ Describe "Import-Alias DRT Unit Tests" -Tags "CI" {
+	BeforeAll {
+		$newLine=[Environment]::NewLine
+
+		$testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ImportAliasTestDirectory
+		$aliasFilename = "pesteralias.txt"
+		$aliasFilenameMoreThanFourValues = "aliasFileMoreThanFourValues.txt"
+		$aliasFilenameLessThanFourValues = "aliasFileLessThanFourValues.txt"
+
+		$aliasfile = Join-Path -Path $testAliasDirectory -ChildPath $aliasFilename
+		$aliasPathMoreThanFourValues = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameMoreThanFourValues
+		$aliasPathLessThanFourValues = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameLessThanFourValues
+
+		$commandToAlias = "echo"
+		$alias1 = "pesterecho"
+		$alias2	= '"abc""def"'
+		$alias3	= '"aaa"'
+		$alias4	= '"a,b"'
+
+		# create alias file
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
 
-		$pesteraliascontent ='# Alias File'+$newLine
-		$pesteraliascontent+='# Exported by : alex'+$newLine
-		$pesteraliascontent+='# Date/Time : Thursday, 12 November 2015 21:55:08'+$newLine
-		$pesteraliascontent+='# Computer : archvm'+$newLine+'"pesterecho","echo","","None"'
-		$pesteraliascontent > $pesteraliasfile
+		# set header
+		$aliasFileContent ='# Alias File'+$newLine
+		$aliasFileContent+='# Exported by : alex'+$newLine
+		$aliasFileContent+='# Date/Time : Thursday, 12 November 2015 21:55:08'+$newLine
+		$aliasFileContent+='# Computer : archvm'
+
+		# add various aliases
+		$aliasFileContent+= $newLine+$alias1+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent+= $newLine+$alias2+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent+= $newLine+$alias3+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent+= $newLine+$alias4+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent > $aliasfile
+
+		# create invalid file with more than four values
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+		$aliasFileContent = $newLine+'"v_1","v_2","v_3","v_4","v_5"'
+		$aliasFileContent > $aliasPathMoreThanFourValues
+
+		# create invalid file with less than four values
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+		$aliasFileContent = $newLine+'"v_1","v_2","v_3"'
+		$aliasFileContent > $aliasPathLessThanFourValues
 	}
 
-	AfterEach {
+	AfterAll {
 		Remove-Item -Path $testAliasDirectory -Recurse -Force
 	}
 
 	It "Should be able to import an alias file successfully" {
-	    { Import-Alias $pesteraliasfile } | Should -Not -throw
+	    {Import-Alias -Path $aliasfile} | Should -Not -Throw
+	}
+
+	It "Should classify an alias as non existent when it is not imported yet" {
+		Get-Alias -Name invalid_alias -ErrorAction SilentlyContinue | Should -BeExactly $null
+	}
+
+	It "Should be able to import an alias file and recognize an imported alias" {
+		Import-Alias -Path $aliasfile
+	    (Get-Alias -Name $alias1 -ErrorAction SilentlyContinue).Definition | Should -BeExactly $commandToAlias
+	}
+
+	It "Should be able to parse ""abc""""def"" into abc""def " {
```

Comment:
There's a bunch of tests here that are very similar in code, you should turn this into using `-TestCases`, it would look something like:

```powershell
It "Should be able to parse <aliasToTest>" -TestCases @(
  @{ aliasToTest = 'abc"def' }
  @{ aliasToTest = 'aaa' }
  @{ aliasToTest = 'a,b' }
) {
   params($aliasToTest)
  Import-Alias -Path $aliasfile
  ( Get-Alias -Name $aliasToTest ).Definition | Should -BeExactly $commandToAlias
}
```

Also, why are you using `-ErrorAction SilentlyContinue`?

Reply:
> There's a bunch of tests here that are very similar in code, you should turn this into using `-TestCases`, it would look something like:
> 
> ```powershell
> It "Should be able to parse <aliasToTest>" -TestCases @(
>   @{ aliasToTest = 'abc"def' }
>   @{ aliasToTest = 'aaa' }
>   @{ aliasToTest = 'a,b' }
> ) {
>    params($aliasToTest)
>   Import-Alias -Path $aliasfile
>   ( Get-Alias -Name $aliasToTest ).Definition | Should -BeExactly $commandToAlias
> }
> ```

Looks good, I'll try that.

> Also, why are you using `-ErrorAction SilentlyContinue`?

iSazonov said [here](https://github.com/PowerShell/PowerShell/pull/9247#discussion_r270524625) that 

> We could check that get-alias -ErrorAction SilentlyContinue returns null before import-alias and returns a value after.

Not using the tag resulted in some local problems, but I recently found out that this was due to local build issues, which I resolved and found out the tag can be removed.